libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -325,24 +421,25 @@ const Review: FC<FieldBaseProps> = ({
             startExpanded={allItemsExpanded}
           >
             <Box paddingY={4}>
-              <GridRow>
-                <GridColumn span="12/12">
-                  <Timeline
-                    initDate={dobDate}
-                    title={formatMessage(m.expectedDateOfBirthTitle)}
-                    titleSmall={formatMessage(m.dateOfBirthTitle)}
-                    periods={formatPeriods(
-                      application.answers.periods as Period[],
-                      otherParentPeriods,
-                    )}
-                  />
-                  <Box paddingTop={3}>
-                    <Button size="small" onClick={() => goToScreen('periods')}>
-                      {formatMessage(mm.leavePlan.change)}
-                    </Button>
-                  </Box>
-                </GridColumn>
-              </GridRow>
+              <Timeline
+                initDate={dobDate}
+                title={formatMessage(m.expectedDateOfBirthTitle)}
+                titleSmall={formatMessage(m.dateOfBirthTitle)}
+                periods={formatPeriods(
+                  application.answers.periods as Period[],
+                  otherParentPeriods,
+                )}
+              />
+              {editable && (
+                <Box paddingTop={3}>
+                  <Button
+                    size="small"
+                    onClick={() => goToScreen && goToScreen('periods')}
```

Comment:
You can use TS optional chaining for function as well
```suggestion
                    onClick={() => goToScreen?.('periods')}
```

Reply:
Oh nice! Will update